### ISSUE TITLE
ble doesn't work on recompile.

### ISSUE THREAD

**Issue by mogorman** [evothings/evothings-viewer#17]
I tried the redbearlab serial example with the application available via the google play store and it works great.  I thien tried to make my own cordova app of it but cannot seem to get ble working.  After trying many things I tried to just build the viewer app locally and then see if that works.  It does not.  This is on an android nexus 6.  I am not sure what other details I could provide but some guidance would be helpful.

**Comment by mogorman** [comment]
It looks like some of my problems were related to having incomplete build environment on my mac.  I needed to install libstd32++c and libz321 on my debian box to get it to build.   I have succeeded in building master branch now, but still do not have working ble support once loaded.

**Comment by mogorman** [comment]
I was able to get everything building.  I have also switched to oracle jdk to be more like what you guys use to build.  i have at least one working plugin as the accelerometer code works in my compile of the viewer. ble still does not.

I tried running the ble scan code, and then going into the debugger in evothings-studio.

```
console.log("im doing it");
test_mog = function(device,errorCode)
{
    if (device) {
        console.log('found + ' + device);
    } else if (errorCode) {
        console.log('error + ' + errorCode);
    }
};
 
app.startScan(test_mog);
console.log('scanning');
```


all i see in the log is im doing it and then scanning.  no error messages.
is there anything else i should be trying?
